LICENSE-3rdparty.csv

```diff
@@ -1041,4 +1041,4 @@ sigs.k8s.io/structured-merge-diff/v4,sigs.k8s.io/structured-merge-diff/v4/fieldp
 sigs.k8s.io/structured-merge-diff/v4,sigs.k8s.io/structured-merge-diff/v4/schema,Apache-2.0
 sigs.k8s.io/structured-merge-diff/v4,sigs.k8s.io/structured-merge-diff/v4/typed,Apache-2.0
 sigs.k8s.io/structured-merge-diff/v4,sigs.k8s.io/structured-merge-diff/v4/value,Apache-2.0
-sigs.k8s.io/yaml,sigs.k8s.io/yaml,MIT
+sigs.k8s.io/yaml,sigs.k8s.io/yaml,Apache-2.0

```

Comment:
A bit confused, because https://github.com/kubernetes-sigs/yaml/blob/master/LICENSE shows this should be MIT, did you manually edit this, or did `make license-check` come up with this?

Reply:
I don't update the file manually and to be honest I don't really know why the license has changed...